packages/cip30/src/BrowserExtension/handleMessages.ts

```
@@ -0,0 +1,14 @@
+import browser from 'webextension-polyfill';
+import { WalletApi } from '..';
+import { Message } from './Message';
+
+export const handleMessages = (walletApi: WalletApi): void => {
+  browser.runtime.onMessage.addListener(async (msg: Message) => {
+    const walletMethod = walletApi[msg.method];
+
+    if (!walletMethod) return; // todo: what to do if method does not exist on WalletApi?
```

Silently ignoring is fine IMO